www/adad05/sp/components/administration.php

```diff
@@ -0,0 +1,142 @@
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+            </select>
+            <button type="submit">Změnit oprávnění</button>
+        </form>
+        <form class="form-admin"  method="post" action="change-user-hotel.php">
+            <label>Změnit hotel uživatele:</label>
+            <select required name="user_id">
+                <?php foreach ($users as $user): ?>
+                    <option value="<?php echo $user['user_id']; ?>"><?php echo $user['email']; ?></option>
+                <?php endforeach; ?>
+            </select>
+            <select required name="hotel_id">
+                <?php foreach ($hotels as $hotel): ?>
+                    <option value="<?php echo $hotel['hotel_id']; ?>"><?php echo $hotel['name']; ?></option>
+                <?php endforeach; ?>
+            </select>
+            <button type="submit">Změnit hotel</button>
+        </form>
+    </div>
+
+    <h2>Správa hotelů</h2>
+    <table class="users-table">
+        <tr>
+            <th>ID</th>
+            <th>Name</th>
+            <th>Address</th>
+        </tr>
+        <?php foreach ($hotels as $hotel): ?>
+            <tr>
+                <td><?php echo $hotel['hotel_id']; ?></td>
+                <td><?php echo $hotel['name']; ?></td>
+                <td><?php echo $hotel['address']; ?></td>
+            </tr>
+        <?php endforeach; ?>
+    </table>
+    <div class="div-update">
+        <form class="form-admin" method="post" action="add-hotel.php">
+            <label>Přidat hotel:</label>
+            <input type="text" min="1" max="80" required placeholder="Název hotelu" name="name">
+            <input type="text" min="1" max="80" required placeholder="Adresa hotelu" name="address">
+            <button type="submit">Přidat hotel</button>
+        </form>
+    </div>
+
+    <h2>Správa vozidel</h2>
+    <table class="users-table">
+        <tr>
+            <th>ID</th>
+            <th>Model</th>
+            <th>Kapacita</th>
+        </tr>
+        <?php foreach ($cars as $car): ?>
+            <tr>
+                <td><?php echo $car['car_id']; ?></td>
+                <td><?php echo $car['model']; ?></td>
+                <td><?php echo $car['capacity']; ?></td>
+            </tr>
+        <?php endforeach; ?>
+    </table>
+    <div class="div-update">
+        <form class="form-admin" method="post" action="add-car.php">
+            <label>Přidat vozidlo:</label>
+            <input type="text" min="1" max="40" required placeholder="Název vozidla" name="model">
+            <input type="number" min="1" max="20" required placeholder="Místa" name="capacity">
+            <button type="submit">Přidat vozidlo</button>
+        </form>
+        <form class="form-admin" method="post" action="change-capacity.php">
+            <label>Změnit kapacitu vozidla:</label>
+            <select required name="car_id">

```

Comment:
ma pocet mist nejaky vliv na rezervaci?

Reply:
Nemá to vliv na logiku, má to funkci pouze informační. V rezervačním okně jsem to nyní přidal k názvu auta.
![image](https://github.com/nvbach91/4IZ278-2023-2024-LS/assets/138229177/d844fe21-e3d3-4d4e-8d3a-1725dc03f301)